### ISSUE TITLE
NewCops: enable doesn't work

### ISSUE THREAD

**Issue by ecylmz** [rubocop-hq/rubocop#8234]
The `bundle exec rubocop --enable-pending-cops` command is running as expected. However, the `NewCops: enable` setting does not `enable` cops with a status of `pending`.

My `.rubocop.yml` file: https://github.com/omu/nokul/blob/rubocop/.rubocop.yml
Inherit from: https://raw.githubusercontent.com/omu/omu/master/etc/rubocop/rubocop.yml

--------

## Expected behavior

Rubocop should work as expected when the following settings are added.

```yml
AllCops:
  NewCops: enable
```

## Actual behavior

`NewCops: enable` does not work. Rubocop continues to warn as follows.

```
The following cops were added to RuboCop, but are not configured. Please set Enabled to either `true` or `false` in your `.rubocop.yml` file:
 - Layout/EmptyLinesAroundAttributeAccessor (0.83)
 - Layout/SpaceAroundMethodCallOperator (0.82)
 - Lint/DeprecatedOpenSSLConstant (0.84)
 - Lint/MixedRegexpCaptureTypes (0.85)
 - Lint/RaiseException (0.81)
 - Lint/StructNewOverride (0.81)
 - Style/ExponentialNotation (0.82)
 - Style/HashEachMethods (0.80)
 - Style/HashTransformKeys (0.80)
 - Style/HashTransformValues (0.80)
 - Style/RedundantFetchBlock (0.86)
 - Style/RedundantRegexpCharacterClass (0.85)
 - Style/RedundantRegexpEscape (0.85)
 - Style/SlicingWithRange (0.83)
For more information: https://docs.rubocop.org/rubocop/versioning.html
```

## Steps to reproduce the problem

Clone the repo with `rubocop` branch: [nokul](https://github.com/omu/nokul/tree/rubocop)

```sh
$ git clone -b rubocop https://github.com/omu/nokul
$ cd nokul
$ rubocop
```

## RuboCop version

```
$ bundle exec rubocop -V
0.86.0 (using Parser 2.7.1.4, rubocop-ast 0.1.0, running on ruby 2.7.0 x86_64-linux)
```

**Comment by ecylmz** [comment]
This setting was not working because incorrect rubocop.yml files remained in the subdirectories.
